blockchain_connector/avalon_blockchain_connector/ethereum/ethereum_connector.py

```diff
@@ -41,7 +42,7 @@
 logging.basicConfig(
     format="%(asctime)s - %(levelname)s - %(message)s", level=logging.INFO)
 
-LISTENER_SLEEP_DURATION = 5  # seconds
+WORKER_SYNCUP_SLEEP_DURATION = 5  # seconds
```

Comment:
Can we put this duration in config file?

Reply:
It's more of an implementation construct. Hence it's not worth exposing it to be configured from outside.

Reply:
makes sense.